shelley/chain-and-ledger/shelley-spec-ledger-test/src/Test/Shelley/Spec/Ledger/Serialisation/EraIndepGenerators.hs

```diff
@@ -955,4 +956,4 @@ instance
   Mock crypto =>
   Arbitrary (Pulser crypto)
   where
-  arbitrary = RSLP <$> arbitrary <*> arbitrary <*> arbitrary <*> arbitrary
+  arbitrary =  RSLP  <$> arbitrary <*> arbitrary <*> arbitrary <*> (RewardAns <$> arbitrary <*> arbitrary)
```

Comment:
I don't know why I apparently added those spaces around `RSLP`

Reply:
I was assuming ormolu would catch such things, but it seems the alonzo tests have escaped the gaze of ormolu...

Reply:
removed now